src/components/Dashboard/TodoList.vue

```diff
@@ -1520,6 +1510,20 @@ export default class TodoList extends Vue {
         console.log('doResumeFiling(), invalid type for item =', item)
         break
     }
+
+    function navigateToCorrectionEditUi (editUrl: string, identifier: string): void {
+      // resume correction via Edit UI
+      const correctionUrl = `${editUrl}${identifier}/correction/?correction-id=${item.filingId}`
+      navigate(correctionUrl)
+    }
+
+    function routeToLocalCorrection (router: VueRouter): void {
+      // resume local correction
+      router.push({
```

Comment:
Can you use `this$router` here to simplify things?

Reply:
I'm unable to access the router inside the function (reading properties of undefined).

Reply:
OK, fine.